portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,73 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sarah Zou's STEP  Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Sarah Zou's Portfolio</h1>
       <p>This is my portfolio.</p>
+      <p>A few quick links: </p>
+      <ul> 
+          <li><a href="#intro">Introduction </a> </li>
```

Comment:
Is there a reason these are indented differently?

Reply:
I wanted the list items to be within the unordered list brackets. I believe that is the accepted formatting for lists in HTML but I'm not 100% sure.